pkg/config/load.go

```diff
@@ -120,6 +120,10 @@ func isConfigFile(info fs.DirEntry) bool {
 	return !info.IsDir() && (extension == ".yaml" || extension == ".yml")
 }
 
+func isMountSpecialFile(path string) bool {
```

Comment:
We probably want to copy the comment that explains k8s configmap behavior to  this func?

Reply:
Added a comment with the general structure of a mount.

Reply:
(this can be handled better, but… one problem at a time)